modules/images-using-images-s2i-php-pulling-images.adoc

```diff
@@ -0,0 +1,58 @@
+// Module included in the following assemblies:
+//
+// * openshift_images/using_images/using-images-source-to-image.adoc
+
+[id="images-using-images-s2i-php-pulling-images_{context}"]
+= Pulling images for PHP
+
+These images come in two options:
+
+* RHEL 8
+* CentOS 7
+
+The RHEL 8 images are available through the Red Hat Registry.
+
+.Procedure
+
+* To pull the RHEL 8 image, enter the following command for the version of PHP you want:
+
+.PHP `5.5`
+[source,terminal]
+----
+$ podman pull registry.redhat.io/openshift3/php-55-rhel7
+----
++
+.PHP `5.6`
+[source,terminal]
+----
+$ podman pull registry.redhat.io/rhscl/php-56-rhel7
+----
++
+.PHP `7.0`
+[source,terminal]
+----
+$ podman pull registry.redhat.io/rhscl/php-70-rhel7
+----
+
+*CentOS 7 Based Images*
+
+CentOS images for PHP 5.5 and 5.6 are available on link:quay.io[Quay.io].
+
+.Procedure
+
+* To pull the CentOS 7 image, enter the following command for the version of Node.js you want:
++
+.PHP `5.5`
+[source,terminal]
+----
+$ podman pull openshift/php-55-centos7
+----
++
+.PHP `5.6`
+[source,terminal]
+----
+$ podman pull openshift/php-56-centos7
+----
```

Comment:
We use registry.redhat.io/ubi8/php-72:latest and registry.redhat.io/rhscl/php-73-rhel7:latest now